src/data/static-class-model.ts

```diff
@@ -0,0 +1,19 @@
+import type { ClassSkillId } from "@/enums/class-skill-id"
+import type { ClassView } from "@/models/character/class-view"
+
+export interface ClassJsonModel {
+    name: string,
+    passives: {
+        modifier: {
+            stat: "health" | "mind" | "inventory" | "meleeAccuracy" | "rangedAccuracy" | "defense" | "magicDefense" | "initiative" | "meleeDamage" | "rangedDamage" | "spellDamage" | "spellAccuracy",
```

Comment:
Make this an Enum or Type in a seperate file to share it with character-class.ts